src/ds/ds-private.h

```diff
@@ -529,6 +519,23 @@ namespace librealsense
             motion_module_fw_version_offset = 212
         };
 
+        enum class gvd_sc_fields      // gvd fields for Safety Camera
```

Comment:
Can we place it in ds6_private? it is not ds common according to the comment..

Reply:
done